webapp/src/components/github_issue/index.tsx

```diff
@@ -114,7 +104,7 @@ const getStyle = makeStyleFromTheme((theme) => ({
         color: theme.buttonColor,
     },
     close_or_reopen_button: {
-        backgroundColor: theme.buttonBg,
+        backgroundColor: '#dc3545',
```

Comment:
Why did you change this?

Reply:
Basically this is the default color of danger.

https://github.com/Brightscout/mattermost-plugin-github/blob/0a8db2e5b68badf7d0d26400a18ce9412f398f36/webapp/src/components/github_issue/index.tsx#L43

Reply:
I got an `errorTextColor` similar to this colour so used that.